db/re/item_db.txt

```diff
@@ -10331,6 +10358,7 @@
 19834,C_Baseball_Cap,Costume Baseball Cap,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,216,{},{},{}
 19835,C_Lif_Doll_Hat,Costume Lif Doll Hat,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,297,{},{},{}
 19836,C_L_Magestic_Goat,Costume Majestic Goat,4,20,,0,,0,,0,0xFFFFFFFF,63,2,1024,,1,0,380,{},{},{}
+19837,C_Asara_Fairy,Costume Asara Fairy Hat,4,10,,0,,,,0,0xFFFFFFFF,63,2,1024,,1,0,492,{},{},{}
```

Comment:
`Position: Upper, Middle` : 3072
All the costumes are sold to 0, can you change yours to 0?

Reply:
@Atemo I didn't understand, why did you mention about the Position?
Which item is wrong?

Reply:
According to https://www.divine-pride.net/database/item/19837
the position of this item is 3072
```
	2^10    1024 = Costume Top Headgear
	2^11    2048 = Costume Mid Headgear
```
It should be
`19837,C_Asara_Fairy,Costume Asara Fairy Hat,4,10,,0,,,,0,0xFFFFFFFF,63,2,3072,,1,0,492,{},{},{}`

Reply:
Ok, I'll fix.
Thanks Atemo!